components/sendbird/sendbird.app.mjs

```diff
@@ -1,11 +1,233 @@
+import SendbirdPlatformSdk from "sendbird-platform-sdk";
+import constants from "./actions/common/constants.mjs";
+
 export default {
   type: "app",
   app: "sendbird",
-  propDefinitions: {},
+  propDefinitions: {
+    channelType: {
+      type: "string",
+      label: "Channel Type",
+      description: "Specifies the type of the channel. Acceptable values are **open_channels** and **group_channels**.",
+      options: constants.CHANNEL_TYPE_OPTS,
+    },
+    applicationId: {
+      type: "string",
+      label: "Application Id",
+      description: "Specifies the unique ID of the Application, To find the application ID, sign in to [Sendbird Dashboard](https://dashboard.sendbird.com/auth/signin), go to **Settings > Application > General**, and then check Application ID.",
+    },
+    channelUrl: {
+      type: "string",
+      label: "Channel URL",
+      description: "Specifies the url of the channel.",
+      async options({ applicationId }) {
+        if (!applicationId) {
+          return [];
+        }
+        const { channels } = await this.listChannels(applicationId);
+        return channels.map((channel) => ({
+          label: channel.name,
+          value: channel.channel_url,
+        }));
+      },
+    },
+    messageTs: {
+      type: "integer",
+      label: "Message Timestamp",
+      description: "Specifies the timestamp to be the reference point of the query in [Unix milliseconds](https://en.wikipedia.org/wiki/Unix_time). Either this or the `Message Id` should be specified.",
+      optional: true,
+    },
+    messageId: {
+      type: "integer",
+      label: "Message Id",
+      description: "Specifies the unique ID of the message to be the reference point of the query. Either this or the `Message Timestamp` should be specified.",
+      optional: true,
+    },
+    prevLimit: {
+      type: "integer",
+      label: "Prev Limit",
+      max: 200,
+      min: 0,
+      description: "Specifies the number of previously sent messages to retrieve before `message_ts`. For example, if `message_ts=1484202848298`, then `prev_limit=50` returns 50 messages sent by 1484202848297 (message_ts - 1).",
+      optional: true,
+    },
+    nextLimit: {
+      type: "integer",
+      label: "Next Limit",
+      max: 200,
+      min: 0,
+      description: "Specifies the number of sent messages to retrieve after `message_ts`. For example, if `message_ts=1484202848298`, then `next_limit=50` returns 50 messages sent from 1484202848299 (message_ts + 1).",
+      optional: true,
+    },
+    include: {
+      type: "boolean",
+      label: "Include",
+      description: "Determines whether to include messages sent exactly on the specified `message_ts` in the results. (Default: **true**)",
```

Comment:
Since the description states a default value, perhaps that should be set as the `default` property of the prop, for example:
```suggestion
      description: "Determines whether to include messages sent exactly on the specified `message_ts` in the results. (Default: **true**)",
      default: true,
```
so as to make use of the Pipedream UI for default values.

Reply:
we cannot do that, because it's going to send the value by default, even tho the property is optional, the API of sendbird can set it as default if it needs, for example, there are properties that cannot be set along another properties, in this case, not set the property is not the same as set an default value even tho it's the same value that they say as its default.